config/application.properties

```diff
@@ -16,3 +16,6 @@ soap.api.base.url=/api/v1
 #soap.api.description=The example REST API in Brahma
 #soap.api.port=10000
 
+#Unit tests configs
+test.restassured.baseUri=http://localhost
```

Comment:
I think we should not load the baseUri from config.  Loading the port is good though.  Could you update this PR?